client/src/app/tabs/dmn/util/configure.js

```diff
@@ -9,7 +9,7 @@
  */
 
 export default function configureModeler(
```

Comment:
Should we consider moving this file to a dmn-shared folder, like we've done for bpmn to eventually down the line refactor dmn and bpmn editors to minimize code duplication, or would you say cloud and platform dmn are too similar to warrant such a change?

Reply:
At the moment, we import directly from the `dmn` folder, so regarding the configure function, this is not really code duplication. I think we can consider having a shared folder in the future, but it's not absolutely necessary at the moment.

Reply:
Alright. with code duplication I was more referring to the DmnEditor classes, like BpmnEditor classes as well. But okay that's not a problem, reducing code duplication in these classes is something I want to have a go at in the future anyways. Maybe a slack time project but you're right it's not absolutely necessary.